packages/flutter_test/lib/src/binding.dart

```diff
@@ -289,9 +289,8 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
 
   /// The value to set [debugCheckIntrinsicSizes] to while tests are running.
   ///
-  /// This can be used to enable additional checks. For example,
-  /// [AutomatedTestWidgetsFlutterBinding] sets this to true, so that all tests
-  /// always run with aggressive intrinsic sizing tests enabled.
+  /// When set to true, tests always run with additional aggressive intrinsic
+  /// sizing tests during layout.
   @protected
   bool get checkIntrinsicSizes => false;
```

Comment:
is this used anymore?

Reply:
This is still used when the binding is initialized in https://github.com/flutter/flutter/blob/d83134ae366e4b2e93c874f7a51260271d88c21a/packages/flutter_test/lib/src/binding.dart#L181-L185

So users have a few ways to set intrinsic sizes (in order of precedence)

1. After the binding is initialzied, set `debugCheckIntrinsicSizes` from `rendering/debug.dart`, either within a test (and resetting the value) or outside of the test
2. Override `checkIntrinsicSizes` on `TestWidgetsFlutterBinding` to a non-null value, then initialize that binding manually
3. Before the test binding is initialized, set `debugCheckIntrinsicSizes` from `rendering/debug.dart`

This is very confusing though, because of duplicated state. I'd suggest removing `checkIntrinsicSizes` from flutter test entirely, and telling users to set the global directly. It [doesn't look like](https://github.com/flutter/flutter/pull/73759) a breaking change, will that be alright?

Reply:
I thought about it a bit more and the two ways of turning on the intrinsic size checks are really not ideal. The latest commit (https://github.com/flutter/flutter/pull/70730/commits/f56b35f1c7200a56cbef7a0e2b3b35bc56c35588) removes `checkIntrinsicSizes` from `TestWidgetsFlutterBinding` to simplify this. If that's not okay let me know and I'll revert it.